src/networks/mod.rs

```diff
@@ -276,7 +276,11 @@ impl ChainConfig {
             f3_enabled: true,
             f3_consensus: true,
             f3_bootstrap_epoch: -1,
-            f3_initial_power_table: None,
+            f3_initial_power_table: Some(
+                "bafy2bzacecklgxd2eksmodvhgurqvorkg3wamgqkrunir3al2gchv2cikgmbu"
```

Comment:
https://github.com/filecoin-project/lotus/blob/v1.32.3/build/buildconstants/params_mainnet.go#L197

```go
// The initial F3 power table CID.
var F3InitialPowerTableCID = cid.MustParse("bafy2bzacecklgxd2eksmodvhgurqvorkg3wamgqkrunir3al2gchv2cikgmbu")
```

Reply:
This is calculated from epoch 4919580 whose states and messages are not included in the latest lite snapshot